src/components/Icon/SeparatorIcon/SeparatorIcon.jsx

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
+import React from 'react';
+import Icon from '../Icon';
+import { lucidClassNames } from '../../../util/style-helpers';
+import { createClass, omitProps } from '../../../util/component-types';
+
+const cx = lucidClassNames.bind('&-SeparatorIcon');
+
+/**
+ *
+ * {"categories": ["visual design", "icons"], "extend": "Icon", "madeFrom": ["Icon"]}
+ *
+ * An Separator icon.
+ */
+const SeparatorIcon = createClass({
+	displayName: 'SeparatorIcon',
+	propTypes: Icon.propTypes,
+
```

Comment:
i would set `_lucidIsPrivate: true` here unless there's another more public use case for this icon